src/coreclr/tools/Common/JitInterface/CorInfoInstructionSet.cs

```diff
@@ -240,54 +239,85 @@ public enum InstructionSet_X86
         X86Serialize_X64 = 40,
     }
 
-    public struct InstructionSetFlags : IEnumerable<InstructionSet>
+    public unsafe struct InstructionSetFlags : IEnumerable<InstructionSet>
     {
-        private ulong _flags;
-
+        private const int _flagsArrSize = 1;
```

Comment:
```suggestion
        private const int FlagsArraySize = 1;
```
Our naming convention for C# constants is to start with upper case and no underscore. Also, we try to avoid abbreviations.